src/utils/jsonwriter.cpp

```diff
@@ -179,10 +179,13 @@ void AsmParser::JsonWriter::writeSource(const asm_line_v *line)
             {
                 this->writeKvNull("file", jsonopt::trailingcomma);
             }
-            else if (line->source.file.starts_with("/app/")) {
+            else if (line->source.file.starts_with("/app/"))
```

Comment:
obviously this is production specific, not sure how to deal with this properly. It also probably shouldn't be in here, but in the parser...